fhd_core/gridding/visibility_grid.pro

```diff
@@ -273,6 +273,12 @@ IF map_flag THEN BEGIN
         *map_fn_inds[i,j]=psf2_inds[psf_dim-i:2*psf_dim-i-1,psf_dim-j:2*psf_dim-j-1]
 ENDIF
 
+if keyword_set(semi_analytic_grid) then begin
+  image_uv=gaussian_grid(vis_arr_use,obs,psf,params,xcen=xcen,ycen=ycen,vis_inds_use=vis_inds_use/n_freq,$
```

Comment:
Please put the `vis_inds_use=vis_inds_use/n_freq` calculation on a new line as a variable, and pass that in. I am a bit concerned about the division by `n_freq` here, and subtle floating-point index errors creeping in.